magma_scripts/positive_rank_twists.m

```diff
@@ -27,11 +27,31 @@ function IsZeroRankofTwist(G,d)
 // It returns true, 0 if all twists of E by the integers d in Dlist indeed have postive
 // algberaic rank. And returns false, d whenever there is an integer d such that E
 // twisted by d has rank 0.
-procedure VerifyPositiveRank(E, Dlist)
+function VerifyPositiveRank(E, Dlist)
     for d in Dlist do
         Ed := QuadraticTwist(E,d);
         if Rank(Ed) eq 0 then
```

Comment:
Could you increase the effort parameter?

Reply:
yep, done, currently running ...